Ladybird/AudioCodecPluginLadybird.cpp

```diff
@@ -339,5 +73,3 @@ Duration AudioCodecPluginLadybird::duration()
 }
 
 }
-
-#include "AudioCodecPluginLadybird.moc"
```

Comment:
I was wondering about this and forgot to go back and check - does Qt only handle this moc stuff automatically if the QObject is in its own header?

Reply:
My understanding is that the cmake infra is smart enough to grep for Q_OBJECT even in .cpp files, but it will only automagically add the moc'd file to the SOURCES list if it was standalone.